components/form/autocompleted/src/index.js

```diff
@@ -66,7 +66,7 @@ export default class FormAutocompleted extends Component {
     })
   }
 
-  _focusInput = () => {
+  focusInput = () => {
```

Comment:
We already use it, but in some cases (such in ours) it's preferable to use a public method as it has less performance implications and re-renderings issues, because we have to manage use cases like this one: https://github.schibsted.io/scmspain/frontend-mt--uilib-components/blob/master/components/search/box/src/index.jsx#L51